gifmodify.js

```diff
@@ -333,6 +393,97 @@ function shiftColor(imgData, index, shiftAmount, randomBlack, randomWhite) {
   return colors;
 }
 
+exports.createWigglingPNG = function(options) {
+  return new Promise((resolve, reject) => {
+    loadImage(options.buffer).then(image => {
+      let size = options.isResized ? 1 : options.size;
+      let height = Math.floor(size * image.height);
+      let width = Math.floor(size * image.width);
+      width += 2 * Math.floor(width / 15); // ~6.6% of width is wiggle room for both sides
+
+      let encoder = new GIFEncoder(width, height);
+      getBuffer(encoder.createReadStream()).then(buffer => resolve(buffer));
+      setEncoderProperties(encoder, options.value * 3);
+
+      addWigglingFramesPNG({
+        image: image,
+        width: width,
+        imgWidth: Math.floor(size * image.width),
```

Comment:
You're doing this twice, rename line 401 to imgWidth and make 402 width. Then change imgWidth here to imgWidth: imgWidth